webauthn-rs-core/src/error.rs

```diff
@@ -274,6 +274,21 @@ pub enum WebauthnError {
 
     #[error("The attestation ca list can not be empty")]
     MissingAttestationCaList,
+
+    #[error("This key has an invalid backup state flag")]
+    SshPublicKeyBackupState,
+
+    #[error("This key is not a valid ssh public key")]
+    SshPublicKeyInvalidType,
+
+    #[error("This requested ssh public key curve is invalid")]
+    SshPublicKeyInvalidCurve,
+
+    #[error("This public key string is not a valid public key")]
```

Comment:
"The SSH public key is invalid"

(though I'm not conflicted as to whether to include the word "string" there)